packages/app/src/systems/Core/utils/machine.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import type { AnyInterpreter, StateFrom, Action } from 'xstate';
+import * as R from 'rambda/immutable';
```

Comment:
We should avoid using new libs when they are not needed. also, rambda even been good. It Would be better to have a more known lib or just write the helpers functions needed in this case.

Reply:
✅